src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-content: center;
+  width: 100px;
+}
+
+.star {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
remove margin-right in last star

Reply:
At first, I made without margin-right in the last star but tests weren't successful. When add it in last star tests were correct.